src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -104,6 +104,9 @@
   <ItemGroup>
     <NuGetPackageToIncludeInVsix Include="StreamJsonRpc" />
     <NuGetPackageToIncludeInVsix Include="Newtonsoft.Json" />
+    <NuGetPackageToIncludeInVsix Include="Microsoft.VisualStudio.Threading" />
+    <NuGetPackageToIncludeInVsix Include="Microsoft.VisualStudio.Validation" />
```

Comment:
Is something tracking this? Any reason we couldn't just have an AssemblyResolve hook or something to pick it up from the existing location on disk?